lib/dbservice_web/controllers/college_controller.ex

```diff
@@ -0,0 +1,163 @@
+defmodule DbserviceWeb.CollegeController do
+  @moduledoc """
+  Handles HTTP requests for College resources.
+  """
+  use DbserviceWeb, :controller
+  use PhoenixSwagger
+
+  alias Dbservice.Colleges.College
+  alias Dbservice.Repo
+  alias DbserviceWeb.SwaggerSchema.College, as: CollegeSchema
+  alias DbserviceWeb.SwaggerSchema.Common, as: SwaggerSchemaCommon
+
+  action_fallback DbserviceWeb.FallbackController
+
+  @doc """
+  Returns the swagger definitions for the College schemas.
+  """
+  def swagger_definitions do
+    Map.merge(
+      CollegeSchema.college(),
+      SwaggerSchemaCommon.group_ids()
+    )
+  end
+
+  @doc """
+  Swagger documentation for the index action.
+  """
+  swagger_path :index do
+    get "/api/colleges"
```

Comment:
why calling it `collges` instead of `college`? just to maintain consistency, for schools it is `/school` instead of `/schools` na?